Sources/Public/Animation/LottiePlaybackMode.swift

```diff
@@ -3,115 +3,155 @@
+    /// - Parameter to: The end progress of the animation.
+    /// - Parameter loopMode: The loop behavior of the animation.
+    case progress(from: AnimationProgressTime? = nil, to: AnimationProgressTime, loopMode: LottieLoopMode)
+
+    /// The animation plays from the given `fromFrame` to the given `toFrame`.
+    /// - Parameter from: The start frame of the animation. If `nil` the animation will start at the current frame.
+    /// - Parameter to: The end frame of the animation.
+    /// - Parameter loopMode: The loop behavior of the animation.
+    case frames(from: AnimationFrameTime? = nil, to: AnimationFrameTime, loopMode: LottieLoopMode)
+
+    /// Plays the animation from a named marker to another marker.
+    ///
+    /// Markers are point in time that are encoded into the Animation data and assigned a name.
+    ///
+    /// NOTE: If markers are not found the play command will exit.
+    ///
+    /// - Parameter from: The start marker for the animation playback. If `nil` the
+    /// animation will start at the current progress.
+    /// - Parameter to: The end marker for the animation playback.
+    /// - Parameter playEndMarkerFrame: A flag to determine whether or not to play the frame of the end marker. If the
+    /// end marker represents the end of the section to play, it should be to true. If the provided end marker
+    /// represents the beginning of the next section, it should be false.
+    /// - Parameter loopMode: The loop behavior of the animation.
+    case markers(from: String? = nil, to: String, playEndMarkerFrame: Bool = true, loopMode: LottieLoopMode)
+
+    /// Plays the animation from a named marker to the end of the marker's duration.
+    ///
+    /// A marker is a point in time with an associated duration that is encoded into the
+    /// animation data and assigned a name.
+    ///
+    /// NOTE: If marker is not found the play command will exit.
+    ///
+    /// - Parameter marker: The start marker for the animation playback.
+    /// - Parameter loopMode: The loop behavior of the animation.
+    case marker(_ name: String, loopMode: LottieLoopMode)
+
+    /// Plays the given markers sequentially in order.
+    ///
+    /// A marker is a point in time with an associated duration that is encoded into the
+    /// animation data and assigned a name. Multiple markers can be played sequentially
+    /// to create programmable animations.
+    ///
+    /// If a marker is not found, it will be skipped.
+    ///
+    /// If a marker doesn't have a duration value, it will play with a duration of 0
+    /// (effectively being skipped).
+    ///
+    /// If another animation is played (by calling any `play` method) while this
+    /// marker sequence is playing, the marker sequence will be cancelled.
+    ///
+    /// - Parameter names: The list of markers to play sequentially.
+    case markerList(_ names: [String])
```

Comment:
Can we keep this as-is?

```suggestion
    case markers(_ names: [String])
```